{"kind":"article","metadata":{"role":"collectionGroup","roleHeading":"API Collection","title":"LocalizedError Implementations","modules":[{"name":"VigramSDK"}]},"identifier":{"url":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/LocalizedError-Implementations","interfaceLanguage":"swift"},"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"topicSections":[{"identifiers":["doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/errorDescription","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/failureReason","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/helpAnchor","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/recoverySuggestion"],"title":"Instance Properties","anchor":"Instance-Properties","generated":true}],"variants":[{"paths":["\/documentation\/vigramsdk\/authenticationerror\/localizederror-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"hierarchy":{"paths":[["doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError"]]},"references":{"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError/failureReason":{"kind":"symbol","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/failureReason","role":"symbol","abstract":[],"type":"topic","title":"failureReason","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"failureReason","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":"?","kind":"text"}],"url":"\/documentation\/vigramsdk\/authenticationerror\/failurereason"},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError/helpAnchor":{"identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/helpAnchor","title":"helpAnchor","type":"topic","url":"\/documentation\/vigramsdk\/authenticationerror\/helpanchor","abstract":[],"role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"helpAnchor","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":"?","kind":"text"}],"kind":"symbol"},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError":{"kind":"symbol","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError","role":"symbol","abstract":[{"type":"text","text":"AuthenticationError contains custom failure events when connecting to Authentication process."}],"type":"topic","title":"AuthenticationError","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"AuthenticationError","kind":"identifier"}],"navigatorTitle":[{"kind":"identifier","text":"AuthenticationError"}],"url":"\/documentation\/vigramsdk\/authenticationerror"},"logo.png":{"alt":"Vigram logo.","type":"image","identifier":"logo.png","variants":[{"traits":["1x","light"],"url":"\/images\/com.vigram.VigramSDK\/logo.png"}]},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError/recoverySuggestion":{"kind":"symbol","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/recoverySuggestion","role":"symbol","abstract":[],"type":"topic","title":"recoverySuggestion","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"recoverySuggestion","kind":"identifier"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"url":"\/documentation\/vigramsdk\/authenticationerror\/recoverysuggestion"},"doc://com.vigram.VigramSDK/documentation/VigramSDK":{"role":"collection","abstract":[],"images":[{"identifier":"logo.png","type":"icon"}],"url":"\/documentation\/vigramsdk","kind":"symbol","title":"VigramSDK","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK","type":"topic"},"doc://com.vigram.VigramSDK/documentation/VigramSDK/AuthenticationError/errorDescription":{"identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/AuthenticationError\/errorDescription","title":"errorDescription","type":"topic","url":"\/documentation\/vigramsdk\/authenticationerror\/errordescription","abstract":[],"role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"errorDescription"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"kind":"text","text":"?"}],"kind":"symbol"}}}